imageprocessor-api/src/main/java/pl/edu/uj/kimage/api/Task.java

```diff
@@ -0,0 +1,42 @@
+package pl.edu.uj.kimage.api;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import java.util.List;
+
+public class Task  implements Serializable {
```

Comment:
It's really important classes (Task, StepDependency, Step, ProcessingSchedule) we don't have much documentation about general idea. We didn't talk about concrete model and I don't understand the purpose of this class.